### ISSUE TITLE
test_could_install_theme is failing due to incorrectly chosen theme

### ISSUE THREAD

**Issue by m8ttyB** [mozilla/Addon-Tests#773]
The test appears to be failing due to this version of Firefox not being supported by this theme. Suggested remedy, ensure the test correctly filters on only themes that are supported by the browser version running the test cases.

amo.stage.saucelabs - https://webqa-ci.mozilla.com/job/amo.stage.saucelabs/241/
    
    tests/desktop/test_installs.py:25: in test_could_install_theme
    Assert.true(complete_theme_page.install_button_exists)
    .env/lib/python2.7/site-packages/unittestzero.py:79: in true
    assert bool(first) is True, msg
        E   AssertionError: None

/cc @krupa

**Comment by bobsilverberg** [comment]
It turns out the problem is that the test is reporting the install button as not visible because there are two install buttons that match the css selector, and the first one is hidden. The first button is for the linux platform and the second one if for windows.  Here is a snippet of the html for the relevant buttons:

```html
      <div class="install-shell">
        <div data-compat-overrides="[]" data-is-compatible-app="true" data-is-compatible="false" data-max="39.*"
             data-min="37.0" data-name="FT DeepDark" data-versions="/en-US/firefox/addon/295337/versions/"
             data-developers="/en-US/firefox/addon/ft-deepdark/developers"
             data-icon="https://addons-stage-cdn.allizom.org/user-media/addon_icons/295/295337-32.png?modified=1430483964"
             data-addon="295337" class="install featuredaddon clickHijack" data-version-supported="true">
          <p class="install-button">
            <a href="/firefox/downloads/latest/295337/platform:2/addon-295337-latest.xpi?src=cb-dl-users"
               data-hash="sha256:e32a62cf8f87b110dd3ae3b0e6f1ad2179d8a853daa16ea50fcf0925ec8685f6"
               class="button  prominent platform linux add installer">
              <b></b>
              <span>Add to Firefox</span>
            </a>
            <a href="/firefox/downloads/latest/295337/platform:5/addon-295337-latest.xpi?src=cb-dl-users"
               data-hash="sha256:e32a62cf8f87b110dd3ae3b0e6f1ad2179d8a853daa16ea50fcf0925ec8685f6"
               class="button  prominent platform windows add installer">
              <b></b>
              <span>Add to Firefox</span>
            </a>
          </p>
        </div>
```

The challenge, looking at this, is that the only thing I can see that differentiates them is the platform name as a class on the `a` tag. I don't really like the idea of hardcoding the platform into the css selector that we are using for the element, as that would mean it would only work on that platform, but I'm not sure what else to do in this instance.

Does anyone have any suggestions? @davehunt?

**Comment by bobsilverberg** [comment]
Actually, I think I have a solution. It's not perfect, but I think it's acceptable. I will submit a PR for it.

**Comment by davehunt** [comment]
Yes, we've seen this in the Mozmill tests in the past for installing addons. I believe we solved it in a similar way (return the first visible install button).

**Comment by bobsilverberg** [comment]
It looks like this was fixed via PR #780 https://github.com/bobsilverberg/Addon-Tests/commit/b0527bc73a557b3f0048dd00c8c96d1c3a4586bb
